integration-tests/src/test/java/com/datastax/oss/driver/api/core/config/DriverConfigValidationIT.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.driver.api.core.config;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import com.datastax.oss.driver.api.core.DriverExecutionException;
+import com.datastax.oss.driver.api.testinfra.session.SessionUtils;
+import com.datastax.oss.driver.api.testinfra.simulacron.SimulacronRule;
+import com.datastax.oss.driver.categories.ParallelizableTests;
+import com.datastax.oss.simulacron.common.cluster.ClusterSpec;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.rules.ExpectedException;
+
+@Category(ParallelizableTests.class)
+public class DriverConfigValidationIT {
+
+  @Rule public SimulacronRule simulacron = new SimulacronRule(ClusterSpec.builder().withNodes(1));
+  @Rule public ExpectedException thrown = ExpectedException.none();
+
+  @Test
+  public void should_fail_to_init_with_invalid_policy() {
```

Comment:
Nice!

Reply:
For some reason this randomly fails in jenkins:

```
17:09:05 Running com.datastax.oss.driver.api.core.config.DriverConfigValidationIT
17:09:05 Tests run: 2, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 1.178 sec <<< FAILURE! - in com.datastax.oss.driver.api.core.config.DriverConfigValidationIT
17:09:05 should_fail_to_init_with_invalid_policy(com.datastax.oss.driver.api.core.config.DriverConfigValidationIT)  Time elapsed: 0 sec  <<< ERROR!
17:09:05 java.lang.NullPointerException
17:09:05 
17:09:06 
17:09:06 Results :
17:09:06 
17:09:06 Tests in error: 
17:09:06 com.datastax.oss.driver.api.core.config.DriverConfigValidationIT.should_fail_to_init_with_invalid_policy(com.datastax.oss.driver.api.core.config.DriverConfigValidationIT)
17:09:06   Run 1: PASS
17:09:06   Run 2: DriverConfigValidationIT.should_fail_to_init_with_invalid_policy » NullPointer
17:09:06 
```

Unfortunately that's not much detail.  I've seen this happen with `ProtocolVersionInitialNegotiationIT.should_use_explicitly_provided_protocol_version` as well, which doesn't make much sense.  Probably something to do with the parallel runner.  Something for me to look at when I do test clean up, but don't think it should hold this back.